kavol@email.cz writes:

I wanted to use the keymap file /usr/share/keymaps/i386/qwerty/cz.map.gz, 
but I have found this to be qwertz so I think it should be placed in 
/usr/share/keymaps/i386/qwertz/ (or, much better, converted to qwerty - 
I don't understand the masochists who use us-qwerty and cz-qwertz together :-)

http://bugs.gentoo.org/show_bug.cgi?id=19010

Gentoo notes:
We copy the qwerty/cz.map to qwertz/cz.map and then apply this patch ... 
that way we end up with both a qwerty and qwertz version of the cz.map

--- data/keymaps/i386/qwerty/cz.map.old	2004-06-21 00:03:22.000000000 +0200
+++ data/keymaps/i386/qwerty/cz.map	2004-10-24 16:04:44.471056808 +0200
@@ -1,6 +1,9 @@
+# *** toto neni originalni verze souboru:
+# *** upraveno z puvodni QWERTZ na QWERTY + opraveno par preklepu
+#
 # Tato keymapa NENI urcena pro jadro "opatchovane" programem "patch-capslock"
 #
-# klavesova mapa kompatibilni s windows QWERTZ
+# klavesova mapa kompatibilni s windows QWERTY
 #
 # Vytvoril: Richard S~puta, Richard.Sputa@os3.os.ds.mfcr.cz
 #
@@ -11,7 +14,7 @@
 # Links: ftp://ftp.penguin.cz/pub/cestina/klavesnice/console/
 #        - primary ftp archiv
 #        http://www.debian.cz/proj/czkbd/index.php3
-#        - czech keyboard standarts
+#        - czech keyboard standards
 #
 #
 # POZOR: Tato klavesova mapa obsahuje ve skutecnosti 2 (dve) klavesnice
@@ -27,7 +30,7 @@
 #               Navic klavesa "PrintScreen" funguje jako carka a hacek
 #
 #
-# Rozsireni oproti standartni_CESKE_unikodove_klavesnici
+# Rozsireni oproti standardni_CESKE_unikodove_klavesnici
 #         (zde jsou popsany pouze kombinace, jejichz umisteni je diskutabilni)
 #         (popis ostatnich rozsireni je v souboru "readme.txt")
 #        1) AltGr-SPACE => znak "nobreakspace" (neoddelitelna mezera kod 0xa0)
@@ -148,7 +151,7 @@
 #          Linefeed  (primarni) => Control_j
 #
 # vzdalenost mezi zacatky symbolu:
-#          12 znaku (standartne) vyhovuje 80% symbolickych nazvu
+#          12 znaku (standardne) vyhovuje 80% symbolickych nazvu
 #          25 znaku ma symbolicky nazev Meta_Control_bracketright
 #             123456789012 123456789012 123456789012 123456789012 123456789012 
 keycode   1 = Escape       Escape       VoidSymbol   VoidSymbol   VoidSymbol  \
@@ -383,15 +386,15 @@
               VoidSymbol   VoidSymbol   +T           t            VoidSymbol  \
               VoidSymbol   VoidSymbol   VoidSymbol   Meta_T       Meta_t      \
               VoidSymbol
-keycode  21 = +z           Z            VoidSymbol   VoidSymbol   Control_z   \
-              Control_z    VoidSymbol   VoidSymbol   Meta_z       Meta_Z      \
-              VoidSymbol   VoidSymbol   +Z           z            VoidSymbol  \
-              VoidSymbol   VoidSymbol   VoidSymbol   Meta_Z       Meta_z      \
+keycode  21 = +y           Y            VoidSymbol   VoidSymbol   Control_y   \
+              Control_y    VoidSymbol   VoidSymbol   Meta_y       Meta_Y      \
+              VoidSymbol   VoidSymbol   +Y           y            VoidSymbol  \
+              VoidSymbol   VoidSymbol   VoidSymbol   Meta_Z       Meta_y      \
               VoidSymbol                                                      \
-              +z           Z            VoidSymbol   VoidSymbol   Control_z   \
-              Control_z    VoidSymbol   VoidSymbol   Meta_z       Meta_Z      \
-              VoidSymbol   VoidSymbol   +Z           z            VoidSymbol  \
-              VoidSymbol   VoidSymbol   VoidSymbol   Meta_Z       Meta_z      \
+              +y           Y            VoidSymbol   VoidSymbol   Control_y   \
+              Control_y    VoidSymbol   VoidSymbol   Meta_y       Meta_Y      \
+              VoidSymbol   VoidSymbol   +Y           y            VoidSymbol  \
+              VoidSymbol   VoidSymbol   VoidSymbol   Meta_Y       Meta_y      \
               VoidSymbol
 keycode  22 = +u           U            VoidSymbol   VoidSymbol   Control_u   \
               Control_u    VoidSymbol   VoidSymbol   Meta_u       Meta_U      \
@@ -601,15 +604,15 @@
               Meta_Control_backslash    bar          bar          VoidSymbol  \
               VoidSymbol   VoidSymbol   VoidSymbol   Meta_bar     Meta_bar    \
               VoidSymbol
-keycode  44 = +y           Y            degree       degree       Control_y   \
-              Control_y    VoidSymbol   VoidSymbol   Meta_y       Meta_Y      \
-              VoidSymbol   VoidSymbol   +Y           y            VoidSymbol  \
-              VoidSymbol   VoidSymbol   VoidSymbol   Meta_Y       Meta_y      \
+keycode  44 = +z           Z            degree       degree       Control_z   \
+              Control_z    VoidSymbol   VoidSymbol   Meta_z       Meta_Z      \
+              VoidSymbol   VoidSymbol   +Z           z            VoidSymbol  \
+              VoidSymbol   VoidSymbol   VoidSymbol   Meta_Z       Meta_z      \
               VoidSymbol                                                      \
-              +y           Y            degree       degree       Control_y   \
-              Control_y    VoidSymbol   VoidSymbol   Meta_y       Meta_Y      \
-              VoidSymbol   VoidSymbol   +Y           y            VoidSymbol  \
-              VoidSymbol   VoidSymbol   VoidSymbol   Meta_Y       Meta_y      \
+              +z           Z            degree       degree       Control_z   \
+              Control_z    VoidSymbol   VoidSymbol   Meta_z       Meta_Z      \
+              VoidSymbol   VoidSymbol   +Z           z            VoidSymbol  \
+              VoidSymbol   VoidSymbol   VoidSymbol   Meta_Z       Meta_z      \
               VoidSymbol
 keycode  45 = +x           X            numbersign   numbersign   Control_x   \
               Control_x    VoidSymbol   VoidSymbol   Meta_x       Meta_X      \
